tests/test_mei.py

```diff
@@ -30,6 +30,28 @@
 
 #         self.assertTrue(mei.decode('utf-8') == target_mei, msg)
 
+class TestExportMEI(unittest.TestCase):
+    def test_export_mei(self):
+        import_score = load_mei(EXAMPLE_MEI)
+        ina = import_score.note_array()
+        with TemporaryDirectory() as tmpdir:
+            tmp_mei = os.path.join(tmpdir, "test.mei")
+            save_mei(import_score, tmp_mei)
+            export_score = load_mei(tmp_mei)
+            ena = export_score.note_array()
+            self.assertTrue(np.all(ina["onset_beat"] == ena["onset_beat"]))
+            self.assertTrue(np.all(ina["duration_beat"] == ena["duration_beat"]))
+            self.assertTrue(np.all(ina["pitch"] == ena["pitch"]))
+            self.assertTrue(np.all(ina["voice"] == ena["voice"]))
+            self.assertTrue(np.all(ina["id"] == ena["id"]))
+
+    def test_export_with_harmony(self):
+        score_fn = os.path.join(MUSICXML_PATH, "test_harmony.musicxml")
+        import_score = load_musicxml(score_fn)
+        with TemporaryDirectory() as tmpdir:
+            tmp_mei = os.path.join(tmpdir, "test.mei")
+            save_mei(import_score, tmp_mei)
+
```

Comment:
I approve this way of testing, but I would add at least one extra test with a different more standard score from our musicxml test scores. The EXAMPLE_MEI is somehow a very weird score

Reply:
I had some problems with making more tests in particular I got some conflicts from MEI import. More of them are related to renaming of the IDs during import. Although the note IDs are consistent in the exported MEI, the IDs during the MEI import change based on a namespace variable (`self.ns`). I will leave it to you if you want to make some changes in the import function.

Reply:
uhm...ok I will look into that!